pkg/aws_config_server/list_roles.go

```diff
@@ -53,13 +56,15 @@ func getWorkerRoles(
 			CredentialsChainVerboseErrors: aws.Bool(true),
 			Retryer:                       session.Config.Retryer,
 		}
-
 		orgClient := awsOrgRoleAssumer(orgAWSConfig)
 		accountList, err := getActiveAccountList(ctx, orgClient)
 		if err != nil {
 			return nil, errors.Wrap(err, "Unable to get list of AWS Profiles")
 		}
 		for _, acct := range accountList {
+			if skipAccts.ContainsElement(*acct.Name) {
```

Comment:
Everything looks great! I was wondering if we should, instead of an account name, use the account ID. That way it is potentially less ambiguous since account names/aliases can change but not account IDs.